src/components/Button/Badge.styles.js

```diff
@@ -1,7 +1,7 @@
-import { StyledButton } from "./Base.styles";
+import { StyledButtonOld } from "./Base.old.styles";
```

Comment:
Instead of keeping "old" styles around. Why don't we introduce a generic "un-styled" button to re-use here and below.

Reply:
The StyledButtonOld name was changed and it was moved to Badge.styles.js. As it is not clear at the moment what styles should be used for Badge, IconButton and ListRowButton according to specs, current styles for these components were left unchanged.

Reply:
let's spend some time to clean this up. Badge & IconButton should have their own stripped down styles. ListRowButton shouldn't really exists. It should be using an updated button based on this work.